src/styles/blocks/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    text-align: center;
+    margin-bottom: 36px;
+  }
```

Comment:
I'm not quite sure that it's allowed to use both inner and outer properties in the same selector.